src/main/java/com/adobe/epubcheck/messages/DefaultSeverities.java

```diff
@@ -136,6 +136,7 @@ private void initialize()
     severities.put(MessageId.HTM_052, Severity.ERROR);
     severities.put(MessageId.HTM_053, Severity.INFO);
     severities.put(MessageId.HTM_054, Severity.ERROR);
+    severities.put(MessageId.HTM_055, Severity.WARNING);
```

Comment:
Oh damn, good point I had not realized it was only informative 🤦‍♂️.

Filed as #1387. 

(I'm not fixing these PRs directly since the code has been released already in the v5.0.0 previews).